src/wallet/wallet2.cpp

```diff
@@ -820,25 +939,25 @@ void wallet2::process_new_transaction(const cryptonote::transaction& tx, const s
 
       BOOST_FOREACH(size_t o, outs)
       {
-	THROW_WALLET_EXCEPTION_IF(tx.vout.size() <= o, error::wallet_internal_error, "wrong out in transaction: internal index=" +
-				  std::to_string(o) + ", total_outs=" + std::to_string(tx.vout.size()));
+        THROW_WALLET_EXCEPTION_IF(tx.vout.size() <= o, error::wallet_internal_error, "wrong out in transaction: internal index=" +
+                                  std::to_string(o) + ", total_outs=" + std::to_string(tx.vout.size()));
```

Comment:
Sorry, I'll undo them.
(I personally like to use spaces instead of tabs for indentation, nvm)